### PR TITLE
NebulaStore can read from follower

### PR DESCRIPTION
**Pull request by panda-sheep**
As title.

### CODE REVIEW

**Review by critical27 — APPROVED** [review]

**Review by liuyu85cn — APPROVED** [review]

File: src/kvstore/plugins/hbase/HBaseStore.cpp [hunk 1]
```diff
@@ -266,8 +270,10 @@ std::pair<ResultCode, std::vector<Status>> HBaseStore::multiGet(
         GraphSpaceID spaceId,
         PartitionID partId,
         const std::vector<std::string>& keys,
-        std::vector<std::string>* values) {
+        std::vector<std::string>* values,
+        bool canReadFromFollower) {
```
Comment (bright-starry-sky) [inline src/kvstore/plugins/hbase/HBaseStore.cpp:274]: If this parameter is not set, there is no need to add `UNUSED`. no big deal.

**Review by bright-starry-sky — APPROVED** [review]
LGTM. Good job.

File: src/kvstore/plugins/hbase/HBaseStore.cpp (hunk 1 above)
Comment (panda-sheep) [reply src/kvstore/plugins/hbase/HBaseStore.cpp:274]: Yeah, you are right.
By the way, is hbase going to be removed?

Comment (bright-starry-sky) [reply src/kvstore/plugins/hbase/HBaseStore.cpp:274]: > Yeah, you are right.
> By the way, is hbase going to be removed?

Yes, hbase will be removed in the future. so no big deal.

Comment (panda-sheep) [reply src/kvstore/plugins/hbase/HBaseStore.cpp:274]: 👌

**Pull request merged**